src/main/java/org/terasology/dynamicCities/settlements/SettlementEntityManager.java

```diff
@@ -132,7 +132,8 @@
 
     private int minDistance = 500;
     private int settlementMaxRadius = 150;
-    private int cyclesLeft = 2; // 1 cycle = approx. 20 seconds
+    private int cyclesLeft; // 1 cycle = approx. 20 seconds
+    private int NUMBER_OF_CYCLES_ON_RESET = 2;
```

Comment:
As pointed out down in `setCityCyclesBeforeGrowth` this member is both read and written, thus, it cannot be a constant. 
As `setCityCyclesBeforeGrowth` seems to be a good name, deriving the member name from the _setter_ method would yield 
```java
    private int cyclesBeforeGrowth = 2;
```

If you want to learn more about constants in Java you may start reading here: 
- https://www.thoughtco.com/constant-2034049
- https://stackoverflow.com/questions/66066/what-is-the-best-way-to-implement-constants-in-java

Reply:
Alright! I got a bit confused about what constants could mean. It all depends on context doesn't it. However, thanks a lot for those resources!. Really helped me clear out some of the concepts for me. I'll go ahead and make that change!